kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -84,338 +80,210 @@ public class KafkaProxyFrontendHandler
         }
     }
 
+    static @NonNull ResponseFrame buildErrorResponseFrame(
```

Comment:
method can be private

Reply:
Hmm interesting. I had at one stage hoped make this available to filter authors to generate errors as well, but as its in terms of `*Frame` its not the right API. A thread to pull on another day...

Reply:
fixed in c840898756cdf924d03a00e37228d26e95b3cf1a